src/ServiceControl.Monitoring.Tests/QueueLength/QueueLengthDataConsumerTests.cs

```diff
@@ -0,0 +1,144 @@
+﻿namespace ServiceControl.Monitoring.Tests.QueueLength
+{
+    using System;
+    using System.Collections.Generic;
+    using Monitoring.QueueLength;
+    using Newtonsoft.Json.Linq;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class QueueLengthDataConsumerTests
+    {
+        [Test]
+        public void It_extracts_sent_sequence()
```

Comment:
Because we said we want to defer the dependency on Metrics here as long as we can. This json is the wire format we accept. Plus it would be quite cumbersome to get that json via metrics and it would not be evident what values are there.